src/libs/SelectionScraper/index.js

```diff
@@ -88,6 +89,12 @@ const replaceNodes = (dom) => {
     let domChildren;
     const domAttribs = {};
 
+    // Encoding HTML chars '< >' in the text, because any HTML will be removed in stripHTML method.
+    if (dom.type === 'text') {
+        // eslint-disable-next-line no-param-reassign
+        dom.data = Str.htmlEncode(dom.data);
```

Comment:
instead of directly modifying the data pass it in the return value.

Reply:
Sorry, I couldn't catch what you need, Could you please explain ?

Reply:
I mean that instead of doing it(violation of rules).

can we do this? 
```js
const data = Str.htmlEncode(dom.data);


....

return {
  ....dom,
 data, 
 };
```

Reply:
done